docs/docs.css

```diff
@@ -0,0 +1,62 @@
+.docs-header {
+  display: flex;
+  flex-direction: column;
+  font-family: "Open Sans", Helvetica, arial, sans-serif;
+}
+
+.docs-header__main {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  padding: 1em 1.25em;
+}
+
+.docs-header__logo {
+  background: url(https://cdn.zapier.com/static/1DZPa0/images/frontend/onboarding/zapier-small-orange-logo.png)
```

Comment:
Q: Will this URL eventually become invalid after a new deploy?

Reply:
That's a very good point @BrunoBernardino. Hmmm. Do we have a more permanent URL for common assets like these?

Reply:
Replied on Slack.